content/2_Bronze/Ad_Hoc.problems.json

```diff
@@ -35,7 +35,7 @@
       "source": "Bronze",
       "difficulty": "Hard",
       "isStarred": false,
-      "tags": [],
+      "tags": ["Implementation"],
```

Comment:
idt this is a tag

Reply:
ig Brute force will work for this one.

Reply:
that's not a tag either

Reply:
its "Complete Search"

Reply:
I have updated it to "Complete Search". Hope it works